src/linter/root_checker.go

```diff
@@ -604,8 +604,32 @@ func (r *rootChecker) checkFuncParam(p *ir.Parameter) {
 		}
 		return true
 	})
-
 	r.CheckTypeHintFunctionParam(p)
+	r.CheckParamNullability(p)
+}
+
+func (r *rootChecker) CheckParamNullability(p *ir.Parameter) {
+	var paramType ir.Node
+	paramType, paramOk := p.VariableType.(*ir.Name)
+	if !paramOk {
+		paramIdentifier, paramIdentifierOk := p.VariableType.(*ir.Identifier)
+		if !paramIdentifierOk {
+			return
+		}
+		paramType = paramIdentifier
+	}
+
+	defValue, defValueOk := p.DefaultValue.(*ir.ConstFetchExpr)
+	if !defValueOk {
+		return
+	}
+
+	if defValue.Constant.Value != "null" {
```

Comment:
`mixed` is an independent type. It already includes `null`
We need to ignore this case.

Reply:
Added